library/src/main/java/com/novoda/downloadmanager/lib/DownloadManager.java

```diff
@@ -1099,8 +1099,8 @@ private long getPausedReason(int status) {
         }
 
         private long getErrorCode(int status) {
-            if ((400 <= status && status < DownloadStatus.MIN_ARTIFICIAL_ERROR_STATUS)
-                    || (500 <= status && status < 600)) {
+            if (400 <= status && status < DownloadStatus.MIN_ARTIFICIAL_ERROR_STATUS
```

Comment:
worth making the effort to boyscout - if you don't know what it means do you know you've not broken anything :)